src/main/java/seedu/weeblingo/ui/FlashcardCard.java

```diff
@@ -45,6 +47,10 @@ public FlashcardCard(Flashcard flashcard, int displayedIndex) {
         this.flashcard = flashcard;
         id.setText("Question: " + displayedIndex + ". ");
         question.setText(flashcard.getQuestion().value);
+        answer.setText(flashcard.getAnswer().value);
+        cardPane.setStyle("-fx-background-image: url('" + image + "'); "
```

Comment:
Perhaps this part can be applied abstraction on? Now the code is getting more and more imperetive haha

Reply:
new commit for added abstraction as well as clearer names for the image variables